src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToStartHandler.java

```diff
@@ -35,8 +46,12 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
 
+        DynamicList languageList = utils.generateInterpreterLanguageFields(sscsCaseData.getAdjournCaseInterpreterLanguage());
+
         clearTransientFields(preSubmitCallbackResponse);
 
```

Comment:
You should try to keep variables close to where they are used:
```suggestion
        clearTransientFields(preSubmitCallbackResponse);
        
        DynamicList languageList = utils.generateInterpreterLanguageFields(sscsCaseData.getAdjournCaseInterpreterLanguage());
```